pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -623,6 +625,74 @@ func getVMIMigrationDataSize(vmi *v1.VirtualMachineInstance) int64 {
 	return memory.ScaledValue(resource.Giga)
 }
 
+func tuneMigrationCompression(vmi *v1.VirtualMachineInstance, params *libvirt.DomainMigrateParameters) {
+	switch compressMethod := vmi.Annotations[v1.LiveMigrationCompressionMethod]; compressMethod {
+	case "mt":
```

Comment:
at some point we should start spliting more logic outside of manager.go.  Maybe at some point we can move all the migration logic to something like `"pkg/virt-launcher/virtwrap/live-migration.go"`. it could still be in the same `virtwrap` package. just a different file.

not necessary for this pr